### PR TITLE
[Backport 2.7] Add cattle-cluster-agent trace/debug logs

### PR DESCRIPTION
**Pull request by felipe-colussi**
This PR Was created to Replace #842.

**Fixes**: There is no issue that this fix.

It adds documentation for the fix created on this issue: https://github.com/rancher/rancher/issues/42241

## Description

Add to the logs doc the possibility of using Environment Variables to get logs out of the cattle-cluster-agent.

### CODE REVIEW

**Review by a-blender — APPROVED** [review]
A few nits but lgtm. I'd suggest `when you initialize downstream clusters` because the phrasing otherwise is odd and sounds like a specific cluster instead of a generic comment about log level. Fix formatting extra line 89, extra space 92

File: docs/troubleshooting/other-troubleshooting-tips/logging.md [hunk 1]
```diff
@@ -78,3 +78,21 @@ possible to enable debug logs in `rancher-machine` when provisioning RKE1, RKE2
 Just like the `trace` log level above, `rancher-machine` debug logs can contain sensitive information.
 
 :::
+
+
+## Cattle-cluster-agent debug logs
+
+The `cattle-cluster-agent` log levels can be set when you initialize the downstream clusters.
```
[the remark this answers is not in the document]
Comment (martyav) [reply docs/troubleshooting/other-troubleshooting-tips/logging.md:85]: Per https://github.com/rancher/rancher-docs/pull/880#pullrequestreview-1644812426

File: docs/troubleshooting/other-troubleshooting-tips/logging.md [hunk 2]
```diff
@@ -78,3 +78,21 @@ possible to enable debug logs in `rancher-machine` when provisioning RKE1, RKE2
...
+
+When you create a cluster under **Cluster Configuration > Agent Environment Vars** you can set variables to define the log level.
+
+
+- Trace-level logging: Set `CATTLE_TRACE` or `RANCHER_TRACE` to `true`
+
+-  Debug-level logging: Set `CATTLE_DEBUG` or `RANCHER_DEUBG` to `true`
```
Comment (martyav) [inline docs/troubleshooting/other-troubleshooting-tips/logging.md:92]:
```suggestion
- Debug-level logging: Set `CATTLE_DEBUG` or `RANCHER_DEUBG` to `true`
```

File: docs/troubleshooting/other-troubleshooting-tips/logging.md [hunk 3]
```diff
@@ -78,3 +78,21 @@ possible to enable debug logs in `rancher-machine` when provisioning RKE1, RKE2
...
 
 :::
+
+
+## Cattle-cluster-agent debug logs
+
+The `cattle-cluster-agent` log levels can be set when you initialize the downstream clusters.
+
+When you create a cluster under **Cluster Configuration > Agent Environment Vars** you can set variables to define the log level.
+
```
Comment (martyav) [inline docs/troubleshooting/other-troubleshooting-tips/logging.md:88]:
```suggestion
```

**Review by martyav — CHANGES_REQUESTED** [review]

File: docs/troubleshooting/other-troubleshooting-tips/logging.md (hunk 2 above)
Comment (btat) [inline docs/troubleshooting/other-troubleshooting-tips/logging.md:92]:
```suggestion
- Debug-level logging: Set `CATTLE_DEBUG` or `RANCHER_DEBUG` to `true`
```

**Review by btat — CHANGES_REQUESTED** [review]
Noticed a typo.

1. Since both the debug and trace levels are set using booleans, does the higher log level take priority when both are true?
2. Assuming 1 is true, what happens in the case where both are enabled and then one of them is subsequently disabled. Is the expected behavior that the log level will be set to the remaining one that's true? E.g. When `CATTLE_TRACE=true, CATTLE_DEBUG=true` becomes `CATTLE_TRACE=false, CATTLE_DEBUG=true` is the log level now at debug?

**Review by btat — APPROVED** [review]

**Review by martyav — APPROVED** [review]

**Pull request merged**